src/xc_integrator/replicated/host/reference_replicated_xc_host_integrator_exc_vxc.hpp

```diff
@@ -223,11 +224,29 @@ void ReferenceReplicatedXCHostIntegrator<ValueType>::
       host_data.vgamma     .resize( gga_dim_scal * npts );
     }
 
+    if( func.is_mgga() ){
+      // TODO: Add check for Laplacian-dependent functionals
```

Comment:
Per previous comment, store this in a variable with the TODO so we can make sure it propagates once we add the logic